api/serverless.ts

```diff
@@ -72,6 +72,7 @@ const dynamicsElevatorSafetyURL = process.env.DYNAMICS_ELEVATOR_SAFETY_URL || ""
 const dynamicsElevatorSafetyClientId = process.env.DYNAMICS_ELEVATOR_SAFETY_CLIENT_ID || "";
 const dynamicsElevatorSafetySecret = process.env.DYNAMICS_ELEVATOR_SAFETY_SECRET || "";
 const dynamicsElevatorSafetyTenantId = process.env.DYNAMICS_ELEVATOR_SAFETY_TENANT_ID || "";
+const featureElevatorOwningBusiness = process.env.FEATURE_ELEVATOR_OWNING_BUSINESS || "";
```

Comment:
Added this feature flag but didn't update any environmental files, because it's all hidden until we get the task approved

Reply:
you may have already done this, but if you haven't, you should add the variable to bitwarden.